### ISSUE TITLE
Issue with traeger integration - Detected code that calls async_forward_entry_setup for integration traeger during setup without awaiting async_forward_entry_setup, which can cause the setup lock to be released before the setup is done. This will stop working in Home Assistant 2025.1.

### ISSUE THREAD

**Issue by drdome** [njobrien1006/hass_traeger#78]
### System Health details

na

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/njobrien1006/hass_traeger/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

The current integration works, but per the log it seems that some updates will need to happen before 2025.1.

### Reproduction steps

1.restart
2.check log
3.
...

### Debug logs

```text
na
```

**Comment by njobrien1006** [comment]
Duplicate of #76

Fixed in current release.

**Comment by drdome** [comment]
Thanks for the quick response.  I thought I was using the latest but apparently the update from August wasn't showing in HA.  I downloaded from GIT and updated manually.  Fixed.
________________________________
